src/navigators/MainTabNavigator.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Icon from '@expo/vector-icons/FontAwesome';
+import { TabNavigator } from 'react-navigation';
+import HomeScreen from '../screens/HomeScreen';
+import ConnectScreen from '../screens/ConnectScreen';
+import VaultScreen from '../screens/VaultScreen';
+import ResponseScreen from '../screens/ResponseScreen';
+import ProfileScreen from '../screens/ProfileScreen';
+import { colors } from '../styles/colors';
+
+
+const MainTabNavigator = TabNavigator({
+  Home: {
+    screen: HomeScreen,
+    navigationOptions: {
+      tabBarLabel: 'Home',
+      tabBarIcon: ({ tintColor }) => (
+        <Icon name="home" size={25} color={tintColor} />
+      ),
+    },
+  },
+  Connect: {
+    screen: ConnectScreen,
+    navigationOptions: {
+      tabBarLabel: 'Connect',
+      tabBarIcon: ({ tintColor }) => (
+        <Icon name="users" size={20} color={tintColor} />
+      ),
+    },
+  },
+  Vault: {
+    screen: VaultScreen,
+    navigationOptions: {
+      tabBarLabel: 'Vault',
+      tabBarIcon: ({ tintColor }) => (
+        <Icon name="briefcase" size={22} color={tintColor} />
+      ),
+    },
+  },
+  Response: {
+    screen: ResponseScreen,
+    navigationOptions: {
+      tabBarLabel: 'Response',
+      tabBarIcon: ({ tintColor }) => (
+        <Icon name="file-o" size={21} color={tintColor} />
+      ),
+    },
+  },
+  Profile: {
+    screen: ProfileScreen,
+    navigationOptions: {
+      tabBarLabel: 'Profile',
+      tabBarIcon: ({ tintColor }) => (
+        <Icon name="user" size={22} color={tintColor} />
+      ),
+    },
+  },
+}, {
+  tabBarPosition: 'bottom',
```

Comment:
are you putting this on the bottom on purpose? `TabNavigator` defaults Android tabbar to the top, which is what people usually expect in an android experience

Reply:
interesting I will have to see what this looks like on android